webpack/src/css/custom/adp.css

```diff
@@ -8752,7 +8775,7 @@ body {
 .has-image {
   width: auto;
   height: auto;
-  background: url("/static/img/header-bg.png");
+  background: url("../../../vitrina/static/img/header-bg.png");
```

Comment:
Use [resolve.alias](https://webpack.js.org/configuration/resolve/#resolvealias) in webpack.config.js. And to the same in other places.